src/browserWindow.ts

```diff
@@ -39,8 +39,7 @@ export class BrowserWindow {
         });
 
         const onDiskPath = path.join(extContext.extensionPath, 'pages/previewPage.html');
-        let data: Buffer = Buffer.alloc(1);
-        data = readFileSync(onDiskPath);
+        let data: string = readFileSync(onDiskPath, 'utf-8');
```

Comment:
```suggestion
        const data: string = readFileSync(onDiskPath, 'utf-8');
```

also appears that `.toString()` below isn't needed anymore